ts/components/wallet/WalletLayout.tsx

```diff
@@ -63,7 +71,12 @@ export default class WalletLayout extends React.Component<Props> {
           </Right>
         </AppHeader>
 
-        <ScrollView bounces={false} style={WalletStyles.whiteBg}>
+        <ScrollView
```

Comment:
This is useless because Content is already a ScrollView (KeyboardAwareScrollView to be precise) and accept all the props of a ScrollView. There is only a typescript typing problem already solved on native-base master branch but still not included in the last release.

Reply:
Can we open another story to act on this change? I tried removing the scrollView and I founded some problems on component rendering to manage

Reply:
Can you aleborate more on the problems you founded?

Reply:
If I remove the scrollview, the upper part of the wallethomescreen overlaps with the botto part (the white one diplaying the list of transactions).